packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/wdio-services/TerraService.tool.mdx

```diff
@@ -55,10 +55,42 @@ export.config = {
             rules: {
               'color-contrast': { enabled: true },
             }
-          }
+          },
         }]
     ],
     // ...
 };
 ```
 
+### formFactor
+
+Tests can be executed in a specific form factor by setting the `formFactor` configuration option or the `FORM_FACTOR` environment variable . The form factors can be any of the following supported viewports: `tiny`; `small`; `medium`; `large`; `huge`; `enormous`. In order for the tests to run in this form factor, it must be one that is already specified in the `Terra.describeViewport` block. 
+
+Type: `String`
+
+Required: `false`
+
+Default: `undefined`
```

Comment:
```suggestion
Default: `huge`
```

Reply:
As discussed in the meeting that we don't want a default here as this is defined by the env variable. Tests will only run in this form factor and not ones defined in the test is this is set.